### PR TITLE
Initial working base

### PR DESCRIPTION
**Pull request by hardyjohnson**
Base cookbook

### CODE REVIEW

File: .kitchen.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+driver:
+  name: vagrant
+
+provisioner:
+  name: chef_solo
```
Comment (dnunez24) [inline .kitchen.yml]: Any reason we wouldn't use `chef_zero` these days?

File: metadata.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name                'cop_base'
+maintainer          'Copious Inc.'
+maintainer_email    'engineering@copiousinc.com'
+license             'MIT'
```
Comment (dnunez24) [inline metadata.rb]: Just an open question about OSL, anyone have an opinion for anything other than MIT? I initially just picked one I felt would be suitable but welcome some dialogue about it.

File: recipes/packages.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+generic_packages = %w(
```
Comment (dnunez24) [inline recipes/packages.rb]: Build as attributes so they can be overridden?

File: metadata.rb (hunk 2 above)
Comment (hardyjohnson) [inline metadata.rb]: MIT is good

**Comment by hardyjohnson** [comment]
Addressed CR comments, please review.

**Pull request merged**